pkg/s3-proxy/server/server.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+const certValidityDuration = 10 * 365 * 24 * time.Hour // 10 years
```

Comment:
Seems to be unused. Can you remove it?

Reply:
Oops, this jumped across files during a refactor. This is needed, but should be in `tls.go`.

Reply:
Oh yes! I didn't see it!

Reply:
Fixed in b904ebe2c7cdf8d153fe1dae7fd22ecce68f780a.